doc/examples.xml

```diff
@@ -709,14 +709,16 @@ true
 <#GAPDoc Label="BookGraph">
 <ManSection>
   <Oper Name="BookGraph" Arg="[filt, ]m"/>
-  <Returns>A digraph.</Returns>
+  <Returns>A graph.</Returns>
```

Comment:
The `Returns` tag should state the type of GAP object that is returned. So here, even though it's mathematically a graph, it is in practice a digraph object that you get back, so this should indeed still say:
```suggestion
  <Returns>A digraph.</Returns>
```